packages/web/src/components/date/DateRange.js

```diff
@@ -61,7 +61,7 @@ class DateRange extends Component {
 	}
 
 	componentDidUpdate(prevProps) {
-		if (!isEqual(this.props.value, prevProps.value)) {
+		if (!isEqual(JSON.stringify(this.props.value), JSON.stringify(prevProps.value))) {
```

Comment:
@anik-ghosh-au7 Why are you using `JSON.stringify`? The `isEqual` method compares the value deeply.

Reply:
The JSON.stringify check was necessary as this was creating the issue, it wasn't recognizing the changes otherwise

Reply:
@anik-ghosh-au7 I can see a separate PR for that. Please remove it from here.

Reply:
@bietkul removed